moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -22,8 +22,8 @@ export const ProjectActivityLogTableMaps = {
         label: "Public description",
         data_type: "text",
       },
-      ecapris_id: {
-        label: "eCapris ID",
+      ecapris_subproject_id: {
+        label: "eCapris Subproject ID",
```

Comment:
In the `newProjectView/DefineProjectForm.js` file the label isn't capitalized. Just noting for consistency: "eCAPRIS subproject ID"

Reply:
Fixed!